about.html

```diff
@@ -0,0 +1,106 @@
+      </ul>
+
+    </nav>
+    
+  </header>
+  <main>
+    <section class="A-first" id="community">
+      <div class="A-intro">
+        <h3>"Hello! It is Here Again"</h3>
+        <h2>NAIROBI INNOVATION WEEK EDITION 3 2022</h2>
+        <p class="short-desc">The inter-continental conference brings togethernthe community of experts, academics and activists who comprise of the Creative Commons affiliate network in a different country every two. This year, we hope to expand our invitation list including organizations and individuals. <span class="long-desc"> who want ot work with us in shared projects that advance the cause of commons free culture and open knowledge. So if you are active and engaged in the worlds of open content and knowledge-free software advocates  Wikipedians, Open knowledge, gallaries, libries, museums, archives, governments and fundations, lawyers and activists- we hope you'll consider joining us this year to build stronger, more vibtrant commons together.</span></p>
+        <p class="event-email">Please contact us per the Email for any further Questions about Nairobi Innovation Week Conference 2022!</p>
+        <p class="email-line"><a href="#">uonbinnovation.ac.ke</a></p>
+      </div>
+    </section>
+    <section class="logo-make" id="Logo">
+        <h2>Nairobi Innovation Week 2022 Logo</h2>
+        <hr class="hr-logo">
+        <p class="para1">The logo for the 2022 Nairobi Innovation week was decided through the logo competition from 1 April to 10.April by the students of the University.After Online pulic voting ad Selection Committee process, the logo from Peter Nguka won the prize.After consultation with Mr Nguka we optimized the prize winning-logo for this year's conference.</p>
+
+        <p class="para2">The logo of the conference 2022 was decided through the logo competition.</p>
+
+        <div class="logo-img">
+          <img src="./imgs/niw2019.jfif" alt="Event Logo">
+        </div>
+    </section>
+
+    <section class="A-third" id="events">
+      <h3>See the past Nairobi Innovation Week</h3>
+      <hr>
+      <p>Take a look at the last two events which took place in Kisumu and Mombasa respectively.</p>
+      <div class="last-events">
+        <a href="https://www.youtube.com/watch?v=-M4SNHmqWG8"><div class="overley"><h3 class="prev-year">2019</h3>
+            <p class="last-para">Nairobi Innonation Week</p></div></a>
+        <a href="https://www.youtube.com/watch?v=-yToP3YiR0I"><div class="overley1"><h3 class="prev-year">2020</h3>
+          <p class="last-para">Nairobi Innonation Week</p></div></a>
+      </div>
+    </section>
+
+    <section class="A-four">
+      <h2>Partner</h2>
+      <hr>
+      <div class="A-sponsors">
+        <img src="./imgs/kcb.png" alt="KCB bank">
+        <img src="./imgs/safaricom.png" alt="Safaricom">
+        <img src="./imgs/microsoft.png" alt="microsoft">
+        <img src="./imgs/udemy.png" alt="Udemy">
+      </div>
```

Comment:
- [ ] [OPTIONAL] You can choose a darker logo for the partner section so your design is as close to the CC design as possible, don't forget to implement this on the home page also

|Your design|CC design|
|-|-|
|![Screenshot_20220528_052128](https://user-images.githubusercontent.com/25227022/170809632-6ccce78b-cca8-49a8-8d22-90b98c2e5ad6.png)|![Screenshot_20220528_052518](https://user-images.githubusercontent.com/25227022/170809696-859b1bb1-b0a9-45aa-a905-d0357546612e.png)|